### DISCUSSION TITLE
[Feature] WEBDAV support

### DISCUSSION THREAD

**Discussion by Majestic7979** [immich-app/immich#3506] [Feature Request]
### The feature

It would be amazing to have my library appear as a folder in my operating system. I could drag files into it to upload new images to Immich, which would then process on the background without having to keep open a browser window (which may crash). And it means I can also browse for my photos from any app as if it's a local folder. Win-win. Should support WebDAV urgently in my opinion, at least it saves you from having to build separate desktop apps for each OS, WebDAV is multiplatform and seamlessly shows up as a folder in the OS.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1687
